FAILED tests/test_cqgi.py::TestCQGI::test_that_invalid_syntax_in_script_fails_immediately - AssertionError: False is not true

Hi, I'm building in Arch Linux today and I have one test failing when building the master branch, commit a6cbf22.
```
==> Starting check()...                                                                                               
================================================ test session starts =================================================
platform linux -- Python 3.10.1, pytest-6.2.5, py-1.11.0, pluggy-0.13.1                                               
rootdir: src/cadquery                                                
plugins: anyio-3.4.0, hypothesis-6.17.0, cov-2.12.1, forked-1.3.0                                                     
collected 439 items                                                                                                   
                                                                                                                      
tests/test_assembly.py .........................................                                               [  9%] 
tests/test_cad_objects.py ...............................                                                      [ 16%] 
tests/test_cadquery.py ....................................................................................... [ 36%] 
....................................................................................                           [ 55%] 
tests/test_cqgi.py ............F...                                                                            [ 58%] 
tests/test_examples.py ......................................................................................  [ 78%] 
tests/test_exporters.py ..........                                                                             [ 80%] 
tests/test_hull.py ..                                                                                          [ 81%] 
tests/test_importers.py ....                                                                                   [ 82%] 
tests/test_jupyter.py ..                                                                                       [ 82%] 
tests/test_selectors.py ...........................................                                            [ 92%] 
tests/test_sketch.py ...............                                                                           [ 95%] 
tests/test_workplanes.py ..................                                                                    [100%] 
                                                                                                                      
====================================================== FAILURES ======================================================
___________________________ TestCQGI.test_that_invalid_syntax_in_script_fails_immediately ____________________________
                                                                                                                      
self = <tests.test_cqgi.TestCQGI testMethod=test_that_invalid_syntax_in_script_fails_immediately>                     
                                                                                                                      
    def test_that_invalid_syntax_in_script_fails_immediately(self):                                                   
        badscript = textwrap.dedent(                                                                                  
            """                                                                                                       
                this doesn't even compile                                                                             
            """                                                                                                       
        )                                                                                                             
                                                                                                                      
        with self.assertRaises(Exception) as context:                                                                 
            model = cqgi.CQModel(badscript)                                                                           
                                                                                                                      
>       self.assertTrue("invalid syntax" in context.exception.args)                                                   
E       AssertionError: False is not true                                                                             
                                                                                                                      
tests/test_cqgi.py:119: AssertionError                                                                                

================================================== warnings summary ==================================================
../../../../../../../../usr/lib/python3.10/site-packages/packaging/version.py:127: 2208 warnings
  /usr/lib/python3.10/site-packages/packaging/version.py:127: DeprecationWarning: Creating a LegacyVersion has been deprecated and will be removed in the next major release
    warnings.warn(

tests/test_cad_objects.py::TestCadObjects::testCompoundCenter
tests/test_cadquery.py::TestCadQuery::testCombineSolidsInLoop
tests/test_cadquery.py::TestCadQuery::testCubePlugin
tests/test_cadquery.py::TestCadQuery::testCylinderPlugin
tests/test_cadquery.py::TestCadQuery::testDoubleTwistedLoft
  src/cadquery/cadquery/utils.py:34: FutureWarning: combineSolids will be removed in the next release.
    warn(f"{f.__name__} will be removed in the next release.", FutureWarning)

tests/test_cadquery.py::TestCadQuery::testCutBlindUntilFace
  src/cadquery/tests/test_cadquery.py:3333: DeprecationWarning: Please use assertAlmostEqual instead.
    self.assertAlmostEquals(wp_ref_regular_cut.val().Volume(), wp.val().Volume())

tests/test_cadquery.py::TestCadQuery::testExtrudeUntilFace
  src/cadquery/tests/test_cadquery.py:3158: DeprecationWarning: Please use assertAlmostEqual instead.
    self.assertAlmostEquals(wp_ref_extrude.val().Volume(), wp.val().Volume())

tests/test_cadquery.py::TestCadQuery::testFindFace
  src/cadquery/cadquery/utils.py:34: FutureWarning: findFace will be removed in the next release.
    warn(f"{f.__name__} will be removed in the next release.", FutureWarning)

tests/test_exporters.py::TestExporters::testAMF
tests/test_exporters.py::TestExporters::testSTEP
tests/test_exporters.py::TestExporters::testSTL
tests/test_exporters.py::TestExporters::testSVG
tests/test_exporters.py::TestExporters::testSVGOptions
tests/test_exporters.py::TestExporters::testTJS
  src/cadquery/cadquery/utils.py:34: FutureWarning: exportShape will be removed in the next release.
    warn(f"{f.__name__} will be removed in the next release.", FutureWarning)

tests/test_exporters.py::TestExporters::testSTEP
tests/test_exporters.py::TestExporters::testSTL
  src/cadquery/cadquery/utils.py:34: FutureWarning: readAndDeleteFile will be removed in the next release.
    warn(f"{f.__name__} will be removed in the next release.", FutureWarning)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============================================== short test summary info ===============================================
FAILED tests/test_cqgi.py::TestCQGI::test_that_invalid_syntax_in_script_fails_immediately - AssertionError: False i...
=================================== 1 failed, 438 passed, 2224 warnings in 44.16s ====================================
==> ERROR: A failure occurred in check().
    Aborting...
```

--- a/tests/test_cqgi.py
+++ b/tests/test_cqgi.py
@@ -113,10 +113,13 @@ class TestCQGI(BaseTest):
             """
         )
 
-        with self.assertRaises(Exception) as context:
-            model = cqgi.CQModel(badscript)
+        xcptn = None
+        try:
+            cqgi.CQModel(badscript)
+        except Exception as e:
+            xcptn = e
 
-        self.assertTrue("invalid syntax" in context.exception.args)
+        self.assertIsInstance(xcptn, SyntaxError)
 
     def test_that_two_results_are_returned(self):
         script = textwrap.dedent(

--- a/tests/test_cqgi.py
+++ b/tests/test_cqgi.py
@@ -113,13 +113,13 @@ class TestCQGI(BaseTest):
             """
         )
 
-        xcptn = None
+        exception = None
         try:
             cqgi.CQModel(badscript)
         except Exception as e:
-            xcptn = e
+            exception = e
 
-        self.assertIsInstance(xcptn, SyntaxError)
+        self.assertIsInstance(exception, SyntaxError)
 
     def test_that_two_results_are_returned(self):
         script = textwrap.dedent(

